logstash-plugins/usagetype-plugin/src/test/java/org/reactome/logstash/plugins/usagetype/TestDataStructures.java

```diff
@@ -0,0 +1,147 @@
+package org.reactome.logstash.plugins.usagetype;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import java.math.BigInteger;
+
+import org.junit.Test;
+import org.reactome.logstash.plugins.usagetype.util.Range;
+import org.reactome.logstash.plugins.usagetype.util.RangeTree;
+
+@SuppressWarnings("static-method")
+public class TestDataStructures
+{
+	@Test
+	public void testRange()
+	{
+		Range<BigInteger> r = new Range<>(BigInteger.valueOf(123), BigInteger.valueOf(456));
+		assertEquals(0,r.isInRange(BigInteger.valueOf(222)));
+		assertTrue(0 > r.isInRange(BigInteger.valueOf(1)));
+		assertTrue(r.isInRange(BigInteger.valueOf(10000)) > 0);
```

Comment:
For the first two lines, why have the constant on the left side of the comparison?

You could do `assertTrue(r.isInRange(BigInteger.valueOf(1) < 0))` for the second line, for example.

Reply:
Yes, I could do that, but I've already done it a different way.

Reply:
I'm fine with either way if you prefer the constant on the left and changing the comparison order of the third line.  

I was just mentioning it for consistency among the test condition pattern for all three lines.